models/quickbooks__general_ledger.sql

```diff
@@ -37,6 +37,13 @@ with gl_union as (
     from {{ref('int_quickbooks__credit_memo_double_entry')}}
     {% endif %}
 
+    {% if var('using_credit_card_payment_txn', False) %}
```

Comment:
The credit card payment source is only used by a few customers. If we add it as true by default it will most likely cause a breaking change for a large number of customers. As such, since only a small portion use this model we will want to have the default be False.

This way they can enable it as True if they are using it and won't break for users that don't use it.

Reply:
Makes sense!